src/dispatch/signal/flows.py

```diff
@@ -33,12 +34,15 @@ def signal_instance_create_flow(
     signal_instance = signal_service.get_signal_instance(
         db_session=db_session, signal_instance_id=signal_instance_id
     )
+    # fetch `all` entities that should be associated with all signal definitions
+    entity_types = entity_type_service.get_all(db_session=db_session, scope="all").all()
```

Comment:
```suggestion
    entity_types = entity_type_service.get_all(db_session=db_session, scope= EntityScopeEnum.all).all()
```

Could use the enum here